MoreLinq.Test/SequenceTest.cs

```diff
@@ -0,0 +1,94 @@
+﻿#region License and Terms
+// MoreLINQ - Extensions to LINQ to Objects
+// Copyright (c) 2016 Leandro F. Vieira (leandromoh). All rights reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#endregion
+
+using NUnit.Framework;
+using System;
+using System.Linq;
```

Comment:
@fsateler True. I didn't pick on that because we need to make sweeping style change there in the test project, where the imports are on the outside (and really harmless because _it is_ a _test project_). In the base project we have them inside the namespace declaration but for sake of consistency, the two should be consolidated to imports inside the namespace. We could [add a style issue](https://github.com/morelinq/MoreLINQ/issues/new).